### ISSUE TITLE
bm02 rebooted

### ISSUE THREAD

**Issue by brucehoff** [Sage-Bionetworks/DigitalMammographyChallenge#545]
```
ubuntu@ip-172-31-59-223:~$ ssh bm02 uptime
 08:46:07 up  8:25,  0 users,  load average: 0.00, 0.01, 0.05
```
Here's the state of the file system after reboot:
```
ssh bm02 df -h
Filesystem                 Size  Used Avail Use% Mounted on
/dev/sda3                   99G   30G   69G  31% /
devtmpfs                   252G     0  252G   0% /dev
tmpfs                      252G     0  252G   0% /dev/shm
tmpfs                      252G   90M  252G   1% /run
tmpfs                      252G     0  252G   0% /sys/fs/cgroup
/dev/mapper/data-dataset2   10T  1.9T  7.6T  20% /data/dataset2
/dev/mapper/data-dataset3   10T  207G  9.3T   3% /data/dataset3
/dev/mapper/data-dataset4   10T  3.0G  9.5T   1% /data/dataset4
/dev/mapper/data-dataset0   10T  2.2T  7.3T  23% /data/dataset0
/dev/mapper/data-dataset1   10T  2.2T  7.3T  23% /data/dataset1
/dev/mapper/data-model1    976M  2.5M  907M   1% /data/model1
/dev/mapper/data-model2    976M  2.5M  907M   1% /data/model2
/dev/mapper/data-model3    976M  2.6M  907M   1% /data/model3
/dev/mapper/data-scratch0  197G  1.1G  186G   1% /data/scratch0
/dev/mapper/data-model0    976M  2.5M  907M   1% /data/model0
/dev/sda1                  248M  216M   32M  88% /boot
/dev/mapper/data-scratch1  197G  1.1G  186G   1% /data/scratch1
/dev/mapper/data-model4    976M  2.6M  907M   1% /data/model4
/dev/mapper/data-data      8.0T  6.6T 1013G  87% /data/data
tmpfs                       51G     0   51G   0% /run/user/1000
```
All containers stopped:
```
CONTAINER ID        IMAGE                                                                                                                         COMMAND                  CREATED             STATUS                    PORTS               NAMES
55bb9e0fc965        docker.synapse.org/syn8091683/dm_docker_pre@sha256:e934b88a8e1f1562ab67e0dbf1e75352c808ad09ff887c3f2d2e27a6a5acda2b           "/preprocess.sh"         23 hours ago        Exited (1) 8 hours ago                        8103820.bm02-2.TRAIN.PREPROCESSING
bfb5c1602551        docker.synapse.org/syn8074722/dm_docker_pre@sha256:e934b88a8e1f1562ab67e0dbf1e75352c808ad09ff887c3f2d2e27a6a5acda2b           "/preprocess.sh"         23 hours ago        Exited (1) 8 hours ago                        8103614.bm02-1.TRAIN.PREPROCESSING
e9794bd222a0        brucehoff/challengedockeragent                                                                                                "/usr/local/bin/mvn-e"   23 hours ago        Exited (1) 8 hours ago                        bm02-train-2
0c4699416f01        brucehoff/challengedockeragent                                                                                                "/usr/local/bin/mvn-e"   23 hours ago        Exited (1) 8 hours ago                        bm02-train-1
```

**Comment by tschaffter** [comment]
@brucehoff We may need to take bm02 out of the Challenge to use for investigating issues. This is currently our top priority.

**Comment by tschaffter** [comment]
The reboot may have been due to the root file system became full at some point. I received the following email alert this morning.

> bm02: FS and Docker data are going to overlap
> File system data are going overlap with the data space reserved for Docker.

With the attached file:

```
FS used (KB): 48369880
FS total (KB): 103492228
Docker used (KB): 24150000.0
Docker total (KB): 80530000.0
```
48 GB (FS used) - 24 GB (Docker used) = 24 GB used by the system, which should never go above 20 GB (100 GB total - 80 GB reserved for Docker). The FS without Docker takes initially 6-7 GB and the email alert is sent when this used space reaches 15 GB.

Current disk usage: 30 GB from `df -h /`
Current Docker data used: 24 GB from `docker info`

The FS used without docker is back to about 6 GB, which is normal.

@brucehoff Do you have any idea why the FS suddenly got 24 GB - 6 GB = 18 GB extra data? Did you transfer pre-processed data?

**Comment by brucehoff** [comment]
> Did you transfer pre-processed data?
No.  I did nothing other than run the standard server-side agents.

**Comment by tschaffter** [comment]
Found them. The 18 GB come from the core dump, which is moved every hour to another partition (`/data/data/crash`). 

```
[root@bm02 127.0.0.1-2017-02-01-00:15:05]# pwd
/data/data/crash/127.0.0.1-2017-02-01-00:15:05
[root@bm02 127.0.0.1-2017-02-01-00:15:05]# du -h --max-depth=0
17G	.
```
Added an entry in crontab so that core dump are also moved after a reboot. Note that core dumps are sometimes created without the machine repeating. For that I keep the script that run every hour.

```
[root@bm02 ~]# last reboot
reboot   system boot  3.10.0-514.2.2.e Wed Feb  1 00:20 - 12:31  (12:10)    
reboot   system boot  3.10.0-514.2.2.e Sat Jan 28 13:36 - 12:31 (3+22:55)
```

`/var/log/messages` don't contain the same crash log than the two issues that we are often facing. Maybe a problem due to critical temperature? I've opened a ticket here: https://ticketing.dst.ibm.com/problem/view/200082
